packages/realm-server/scripts/start-prerender-manager-staging.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/sh
+
+# Start the prerender manager in staging
+
+NODE_ENV=production \
+  NODE_NO_WARNINGS=1 \
+  ts-node \
+  --transpileOnly prerender/manager-server \
+  --port=${PRERENDER_MANAGER_PORT:-4222}
```

Comment:
I guess it follows the realm server pattern but is there a reason to have production and staging scripts if they’re not different?